metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_gca020631705v2_about.md

```diff
@@ -0,0 +1,14 @@
+**About *Daphnia magna***
+-------------------------
+*Daphnia magna* is a small planktonic crustacean that belongs to the 
+subclass Phyllopoda.
```

Comment:
I would be tempted to expand a bit on this organism in terms of description. Perhaps could be useful to add in...

"It inhabits a variety of freshwater environments and is broadly distributed throughout the northern hemisphere, and in South Africa. 

D. magna is used in many different fields of research, such as ecotoxicology, population genetics, the evolution of sex, phenotypic plasticity, ecophysiology (including global change biology) and host-parasite interactions."

Reply:
thanks. done.